plugins/flashrom/fu-reset-cmos.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2021 Sean Rhodes <sean@starlabs.systems>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <fwupdplugin.h>
+
+/* From coreboot's src/include/pc80/mc146818rtc.h file */
+#define RTC_BASE_PORT 0x70
+
+/*
+ * This is the offset of the first of the two checksum bytes
+ * we may want to figure out how we can determine this dynamically
+ * during execution.
+ */
+#define CMOS_CHECKSUM_OFFSET 123
+
+gboolean
+fu_reset_cmos(GError **error);
+
+bool
+cmos_write_byte(guint8 addr, guint8 val);
```

Comment:
you don't want to be exporting this, it should be a static function, it just needs to be declared before it's first use

Reply:
Thank you!!!